### ISSUE TITLE
Starting on collation!

### ISSUE THREAD

**Issue by haggis78** [haggis78/BreconChurch#3]
Amber and Connor --

We now have a rough xml document of the text, taken from the earliest printing, on our repo. The next steps are a) for me to continue pounding out a rough translation, and b) for you to compare this text with the two other printed versions we're looking at. Fuller details are in an xml comment at the top of the document, which I've named BreconChurchCharter.xml.

Are both of you available to meet on Friday at 4?

-Dr. Campbell

**Comment by amberpeddicord** [comment]
@haggis78 Sounds great! And, I should be able to meet at 4 on Friday.

**Comment by ChinoyIndustries** [comment]
Sounds good to me! I can definitely meet after class Friday, I'm free until a meeting at 6 (which I can't imagine would be a problem).

**Comment by haggis78** [comment]
I have created a new folder on Box where I have dropped just the images and files relevant to our project. If you go into the Welsh Fasti Box folder to which you already have access, you will find a folder within that named Brecon_files_for_Box. Again, we're not using Github for these right now because we do not have permission to share these images on the Web yet. (Also, Dr. B said that Github isn't the best way to share high-res images anyway.) The xml file remains on Github, however, where it belongs.

I created a sub-folder for each manuscript we'll be looking at. The first letter of each folder name is the siglum (reference letter) we'll be using to keep track of that manuscript, just like the W, S, and J we are using for the printed witnesses. The remainder of each folder name is the manuscript's reference number in its respective archive.

I also added in the same place the Word file of my partial, rough translation of the text. As you'll see it can be exhaustively repetitive to the point of being almost comical at times. I'll finish it when I get a chance.

How are you doing with the printed versions? If you expect to be finished before the weekend is out, we should probably each take a manuscript at our meeting on Friday so there's something to get started with. I'll be out of town and incommunicado over the weekend, so I won't be able to answer any questions, but even starting to list the _types_ of issues that show up (marginal notes, corrections, deletions, insertions, faded ink, whatever) would give us more of a sense of what we need to think about in terms of a schema/TEI.

**Comment by haggis78** [comment]
@amberpeddicord @ChinoyIndustries -- We now have a unified file to pull, edit, and push, but we need to make sure that we're not working on it at the same time. It's called CharterCollation.xml. I've got dibs on working on it tonight. We need to co-ordinate, so before executing a pull to start working, or a push after working, send up a flare over Github so we'll all know not to create a merge conflict!

Connor, we deleted two previous versions with file names indicating collation with S and W. When you next pull, Github will delete them from the repo on your computer. That's as it should be. When you get a chance to start comparing the text with J, you'll see the pattern Amber has established for us for the variant readings.

Amber, I checked each of the deviations between S and W and I found several that were simply typos in transcribing W that I hadn't caught yet. I deleted the app and rdg tags and just returned it to the unified text as it should have been if I'd been more sharp-eyed. But there are some definite deviations.
